ui/lib/core/addon/utils/client-count-utils.ts

```diff
@@ -78,17 +78,50 @@ export const filterVersionHistory = (
 // This method is used to return totals relevant only to the specified
 // mount path within the specified namespace.
 export const filteredTotalForMount = (
-  byNamespace: ByNamespaceClients[],
+  byMonth: ByMonthClients[],
   nsPath: string,
   mountPath: string
 ): TotalClients => {
-  if (!nsPath || !mountPath || isEmpty(byNamespace)) return emptyCounts();
-  return (
-    byNamespace
-      .find((namespace) => sanitizePath(namespace.label) === sanitizePath(nsPath))
-      ?.mounts.find((mount: MountClients) => sanitizePath(mount.label) === sanitizePath(mountPath)) ||
-    emptyCounts()
-  );
+  if (!nsPath || !mountPath || isEmpty(byMonth)) return emptyCounts();
+
+  return newClientTotal(byMonth, nsPath, mountPath);
+};
+
+export const newClientTotal = (
```

Comment:
The UI should avoid doing any calculations. The `total.clients` in the `/activity` response is the  "total new clients" for a billing period, so this number is already provided for us.

Reply:
The top level `total` object on the response is the running total for that queried date range. So the numbers left of the chart should match the response one to one. 
<img width="395" alt="Screenshot 2025-05-12 at 3 06 53 PM" src="https://github.com/user-attachments/assets/46118ac5-c205-445b-b173-85f3d917c51b" />